webpack/assets/javascripts/react_app/components/statistics/ChartBox.test.js

```diff
@@ -48,4 +48,16 @@ describe('ChartBox', () => {
     box.setState({ showModal: true });
     expect(box.find('Modal').length).toBe(1);
   });
+  it('shouldComponentUpdate should be called', () => {
+    const spy = jest.spyOn(ChartBox.prototype, 'shouldComponentUpdate');
+    const box = shallow(<ChartBox
+      id='4'
+      type="donut"
+      chart={{ id: '4', data: undefined }}
+      status="PENDING"
+    />);
+
+    box.setProps({ status: 'PENDING' });
+    expect(spy).toHaveBeenCalled();
+  });
```

Comment:
does it make sense to add another test that check the shouldComponentUpdate logic as well?